Lesson11/Task4.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Сделайте функцию, которая принимает параметром число от 1 до 7, а возвращает день недели на русском языке.
+ */
+function weekDay($weekDayNumber) {
+    $week = [1 => 'понедельник', 'вторник','среда','четверг','пятница', 'суббота', 'воскресенье'];
+    if ($weekDayNumber >= 1 && $weekDayNumber <= 7) {
+        return $week[$weekDayNumber];
+    } else {
+        return false;
```

Comment:
better to have just one return type in function

Reply:
so change false to 0 pls
0 == false

Reply:
But the return type must be string, not int: $week array has string names of weekDays. It may be done like "" == false.